src/module.ts

```diff
@@ -296,6 +321,48 @@ export const plugin = new PanelPlugin<PanelOptions>(VariablePanel)
         category: ['Layout'],
         showIf: (config) => showForMinimizeView(config) || !config.groups?.length,
       })
+      .addRadio({
+        path: 'minimizeDisplayMode',
```

Comment:
It needs to be considered to simplify 3 options into 2: outputFormat(or another name) and isUseLocalTime.
If the variable type is text or constant then the outputFormat option is shown with the following variants: text, date string, timestamp. isUseLocalTime is shown if date string or timestamp selected

Reply:
updated